make/test-integration.mk

```diff
@@ -1,4 +1,4 @@
-COVER_PACKAGES = $(shell go list ./... | grep -v mock)
+COVER_PACKAGES = $(shell go list ./... | grep -v mock | tr "\n" "," | rev | cut -c2- | rev)
```

Comment:
maybe just [bash substitution](https://gist.github.com/JPvRiel/b279524d3e56229a896477cb8082a72b) then?

Reply:
was having issues making bash substitution work with makefile, so just used the makefile way as initially suggested by you:
> ```makefile
> space := $(subst ,, )
> comma := ,
> 
> test-coverage:
> 	$(GO) test -tags=$(BUILD_MAINNET) -coverprofile=coverage.txt \
> 		-covermode=count \
> 		-coverpkg=$(subst $(space),$(comma),$(COVER_PACKAGES)) \
> 		./...
> ```